charts/artifact-hub/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: artifact-hub
 description: Artifact Hub is a web-based application that enables finding, installing, and publishing Kubernetes packages.
 type: application
-version: 1.5.1-0
+version: 1.5.2-0
```

Comment:
Could you use `1.5.1-1` instead of `1.5.2-0`? Otherwise if we need to release a `1.5.1` version the chart linter will complain and ask us to bump it to a higher version.

Reply:
@tegioz fixed, thank you for the explanation.